### ISSUE TITLE
How to know which packages are official?

### ISSUE THREAD

**Issue by jabbalaci** [nim-lang/nimble#570]
As written in the Nim book, the Nimble packages are split into official and unofficial lists. How to see just the official packages? At https://nimble.directory/ all packages are together.

**Comment by dom96** [comment]
No, PRs are reviewed for that reason.

**Comment by FedericoCeratto** [comment]
@dom96 the package directory is searching for keywords in the description and tags.

The "official" packages are a very small minority. Nimble treats all packages equally at the moment. Without a vetting process for other packages there's little benefit in adding that tag.

**Comment by dom96** [comment]
yeah, to be honest, as much as the "official" concept is good, we're not that great at keeping those packages up to date so it may be better to not add any further "official" tags.
